requests.go

```diff
@@ -609,3 +625,24 @@ func DefaultTransport(tlsNoVerify bool) http.RoundTripper {
 
 	return transport
 }
+
+func TLSTransport(certPEMBlock, keyPEMBlock, caCert []byte) http.RoundTripper {
```

Comment:
This will be missing all of the attributes in the default transport. I'd rather have one function (DefaultTransport) that also accepts these parameters and only &tls.Config is modified. We want both transports to return the same values for all the other attributes.